app/views/accounts/_webauthn_platform.html.erb

```diff
@@ -0,0 +1,33 @@
+<div>
```

Comment:
Do we need this extra wrapper, or can it be removed?

**Edit:** I see now this file is largely copied from and inheriting the existing webauthn view.